public/app/features/users/TokenRevokedModal.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Button, Modal } from '@grafana/ui';
+import config from 'app/core/config';
+
+interface Props {}
+
+interface State {}
+
+export class TokenRevokedModal extends React.Component<Props, State> {
```

Comment:
```suggestion
export const TokenRevokedModal = () => {
```
It can be a simple functional component, plus no need for empty types if it doesn't have any props.

Reply:
Absolutely, I kept them while developing and I forgot to remove them. Thanks!